frontend/src/components/dialogs/SecretDialogGeneric.vue

```diff
@@ -92,7 +92,7 @@ export default {
       const validators = {
         data: {
           required,
-          isYAML: () => isObject(this.parsedYAML)
+          isYAML: () => Object.keys(this.parsedYAML).length
```

Comment:
```suggestion
          isYAML: () => Object.keys(this.parsedYAML).length > 0
```
It should return a boolean if nt directly use inside an if condition. Use `!!` or `> 0`.